terraform/ec2/creds/variables.tf

```diff
@@ -106,3 +106,9 @@ variable "plugin_tests" {
   type    = string
   default = ""
 }
+
+variable "agent_start" {
```

Comment:
Does this `agent_start` variable need to be propagated to the Windows tf as well? I also don't see where these variables are applied or referenced. Is that supposed to be here, or in the main agent repo?

Reply:
nvm found it https://github.com/aws/amazon-cloudwatch-agent/pull/782